src/handler/commandfuture.rs

```diff
@@ -20,6 +20,8 @@ pin_project! {
         rx_command: oneshot::Receiver<M>,
         #[pin]
         target_sender: mpsc::Sender<TargetMessage>,
+        #[pin]
```

Comment:
can we add a oneline doc why we're using this

Reply:
Otherwise we would not be possible to call .poll on this DelayFuture, #[pin] allows us to do that.
Or I didn't found the correct way to do that.